FinSearchUnified/Bundle/SearchBundleFindologic/FacetHandler/TextFacetHandler.php

```diff
@@ -122,6 +122,10 @@ private function getValueListItems(FacetInterface $facet, Criteria $criteria, Si
      */
     private function createValueListFacetResult(FacetInterface $facet, Criteria $criteria, SimpleXMLElement $filter)
     {
+        if (!$filter->items->item) {
+            return null;
+        }
```

Comment:
Yes. It was actually part of one of the test scenarios, and it could only pass if we returned from here otherwise it would throw an error about the items being null. It was not part of the implementation though. Like i said on youtrack there were some missing things in the description which I had to figure out. If something is not correct let me know and I'll make the necessary changes.

Reply:
Looks correct, just wanted to know the reason behind it.